### PR TITLE
Add tests for lazy load

### PR DESCRIPTION
**Pull request by andymost**
Tests that checks lazy load create script element for app after DG.then call, and that for lazy=false create it immediatly

### CODE REVIEW

File: functional-tests/classes/util/scripts.py [hunk 1]
```diff
@@ -22,6 +22,10 @@ def set_lang(lang):
     def open_marker(name='marker'):
         return '%s.openPopup();' % name
 
+    @staticmethod
+    def dg_then(cb=''):
+        return 'DG.then(%s)' % cb
```
Comment (RLRR) [inline functional-tests/classes/util/scripts.py]: Точку с запятой бы поставить)

**Comment by RLRR** [comment]
:+1:

**Comment by andymost** [comment]
@Trufi обновил

**Comment by Trufi** [comment]
:+1:

**Pull request merged**